src/catalog/jackson-bindings/geotools/src/test/java/org/geotools/jackson/databind/filter/model/TestDto.java

```diff
@@ -0,0 +1,15 @@
+package org.geotools.jackson.databind.filter.model;
+
+import java.util.Date;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
```

Comment:
These can be replaced by a single `@Data` annotation, or `@Data` and `@AllArgsConstructor`?